src/musicalbum.rb

```diff
@@ -0,0 +1,52 @@
+require_relative './app'
+require_relative './item'
+require 'date'
+
+class MusicAlbum < Item
```

Comment:
Every music album should have an author, genre and label. However, you are missing all of these things. I have recorded [this video](https://www.loom.com/share/3bb2fbd21a324177b1ba95212895e1bc) where I explained in detail what you need to fix.